install_config/aggregate_logging.adoc

```diff
@@ -2054,6 +2054,186 @@ $ oc exec <$espod> -- es_load_kibana_ui_objects <user-name>
 
 Where `$espod` is the name of any one of your Elasticsearch pods.
 
+[[kibana-visualizations-custom]]
+*Adding Custom Fields to Kibana Visualize*
+
+If your {product-title} cluster is configured with custom logging fields
+that are not are not defined in the `.operations.*` or specific `project.*` 
+index patterns, you cannot create visualizations with these fields. 
+
+However, if you if you use the `MERGE_JSON_LOG` feature (`MERGE_JSON_LOG=true`) 
+to generate JSON format logs, you can use these custom fields in 
+Kibana Visualize by adding the fields to the index patterns and to Kibana. 
+
+To add custom fields to Kibana Visualize:
+
+. Determine which index patterns you want to add the fields to, either
+`.operations.*` or a specific `project.*` index pattern. 
+If there is a specific project that has the custom fields, 
+you can specify a specific index pattern for the indices for the project,
+for example: `project.this-project-has-time-fields.*`.
+
+. Create a JSON file for each index pattern, similar to the following:
++
+For example:
++
+----
+{
+	"order": 20,
+	"mappings": {
+		"_default_": {
+			"properties": {
+				"mytimefield1": {
+					"doc_values": true,
+					"format": "yyyy-MM-dd HH:mm:ss,SSSZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ||yyyy-MM-dd'T'HH:mm:ssZ||dateOptionalTime",
+					"index": "not_analyzed",
+					"type": "date"
+				},
+				"mytimefield2": {
+					"doc_values": true,
+					"format": "yyyy-MM-dd HH:mm:ss,SSSZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ||yyyy-MM-dd'T'HH:mm:ssZ||dateOptionalTime",
+					"index": "not_analyzed",
+					"type": "date"
+				}
+			}
+		}
+	},
+	"template": "project.<project-name>.*" <1>
+}
+----
+<1> Specify the `.operations.*` or a `project.` index pattern.
+
+. Get the name of one of the Elasticsearch pods:
++
+----
+$ oc get -n logging pods -l component=es
+
+NAME                                       READY     STATUS    RESTARTS   AGE       IP             NODE                         NOMINATED NODE
+logging-es-data-master-5av030lk-1-2x494    2/2       Running   0          38m       154.128.0.80   ip-153-12-8-6.wef.internal   <none>
+----
+
+. Load the JSON file into the Elasticsearch pod:
++
+----
+$ cat <json-file-name> | \
+oc exec -n logging -i -c elasticsearch <es-pod-name> -- \
+    curl -s -k --cert /etc/elasticsearch/secret/admin-cert \
+    --key /etc/elasticsearch/secret/admin-key \
+    https://localhost:9200/_template/<json-file-name> -XPUT -d@- | \
+python -mjson.tool
+----
```

Comment:
@QiaolingTang @richm 
What would this command look like to add a JSON to a pod if using project index